lab-goat/model/transform.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
```

Comment:
Looks like you were on the way to modularizing, but duplicated code into this file as part of that process?